### DISCUSSION TITLE
zh/guide/drivers/alias

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#198] [Announcements]
# zh/guide/drivers/alias

我们先看一张示意图 alias 我们看到两个不同的文件夹合并在一个文件夹里面了，名字一样的文件文件夹也都合并在一起了，独有的单独显示了 例1 riluo.jpg 是本地1 独有的，在上面显示了; 例2 两个文件夹都有video文件夹，但是里面的内容也会进行合并，子文件夹也同样适用于同名合并的展示规则; 两个文件夹分别有两个视频，但是有一个是同名的，最后...

https://alist.nn.ci/zh/guide/drivers/alias.html

**Comment by no2coolboy** [comment]
没看明白要怎么设置

**Comment by xjc19960724** [comment]
啥时候支持删除文件呢

**Comment by chux1995** [comment]
编写挂载路径别加中文，我使用infuse的时候发现他没法识别这个路径。

**Comment by jinyu2022** [comment]
完全没看明白

**Comment by chux1995** [comment]
还有一个问题，就是我发现通过这个加载我本地local的数据，速度会慢一倍以上，我从原生的路径读取本地环境文件有50MB/S的速度，切换到这个别名的路径，就不到10MB/S了。设置的朋友可以自行测试一下，我使用infuse挂载webdav和网页端alist都有这个问题，并且程序还崩溃过。

**Comment by ebuact** [comment]
那个大佬能帮忙解释一下这到底说的什么意思，完全没明白

**Comment by Adsryen** [comment]
有应用场景的，一眼就懂，但是我懒得配，我愿意手动多打开一次文件夹没关系

**Comment by Mackerly** [comment]
没明白，Windows下设置，挂载路径：/迅雷下载 
 路径：
本地:D:\迅雷下载
本地:E:\迅雷下载

好像没效果？不知哪里错了

**Comment by Futurescope** [comment]
本地目录挂载别名，能看懂前面但是不懂的是路径为什么要用中文“本地:”开头？也没有联想也没有提示，挂载smb配合symlink弄好了

**Comment by sjtuross** [comment]
实际测试第二种填写方式不起作用，效果却跟第一种填写方式一样

**Comment by ZhangTianrong** [comment]
然而重命名并没有效果……比如我配置路径
```
CloudDownload:/115-AList/
CloudDownload:/PikPak-AList/Pack From Shared/
CloudDownload:/Aliyun-AList/好友的分享/
```
结果驱动下并没有 `CloudDownload` 目录，再添加
```
Storage:/ChinaMobile-AList/AList/
Storage:/ChinaTelecom-AList/
Storage:/Aliyun-AList/
Storage:/PikPak-AList/My Pack/
Storage:/SuperNova-AList/
```
直接覆盖了 `CloudDownload`. 我就是很基础地想要通过全局设置隐藏原始挂载路径，然后用 Alias 聚合显示啊，结果基本处于不可用状态……还是只把 AList 当作一个不怎么可靠的 Webdav backend 然后用 rClone 的 Union 解决吧，要是想用 AList 当前端，那就 `rclone serve <union>: sftp`，然后再当作新的驱动添加进来，还不会遇到这里的只读限制。

**Reply by ZhangTianrong** [reply]
末尾 `/` 加不加都试过，没区别。

**Reply by ZhangTianrong** [reply]
哦，我理解了，这个重命名后的路径就是被聚合的路径，所以只能有一个……不同的路径就用不同的 Alias 搞……行吧。

**Comment by Soltus** [comment]
尝试了一下，聪明的我终于弄懂了（有多余的步骤)！
首先使用`本地存储`方式添加挂载的文件夹，以：`F:\迅雷下载`挂载到`/不显示的挂载路径/迅雷下载路径1`，`G:\迅雷下载`挂载到`/不显示的挂载路径/迅雷下载路径2`为例，然后使用`别名`挂载，`路径`参数如下：
```bash
本地:/不显示的挂载路径/迅雷下载路径1
本地:/不显示的挂载路径/迅雷下载路径2
```
显然，上面的操作添加了三个新的纯粹，将前两个挂载路径隐藏即可

**Comment by se7enmuting** [comment]
使用的下载方式 这块没看明白
我某个别名AAA的路径是：
/NAS-01  （NAS本地磁盘，WebDAV 策略：本地代理）
/aliyunpan   （阿里云盘，WebDAV 策略：302）
然后这个别名AAA的WebDAV 策略：302
那这样我通过这个别名访问NAS-01是用什么WebDAV 策略？
如果我这个别名AAA的WebDAV 策略改成：本地代理，那访问 阿里云盘 时是什么策略？

**Comment by mlkt** [comment]
名字起得不对，应该叫union，但是逻辑有点诡异，建议学习一下rclone union的配置

**Comment by devouer** [comment]
alist别名是不是别名

**Comment by JiuMengDz** [comment]
同没看懂这部分  能不能配个视频配合食用呢？急需这个功能用于分配不同的用户访问不同的文件夹

**Comment by haohao232323** [comment]
刚刚测试了一下，可以合并多个网盘的文件夹，但是假如想穿透子文件夹和孙文件夹不行，我主要是想把重复文件合并一下，例如A网盘-国产剧-《大宅门》，B网盘-影视-电视剧-豆瓣高分系列-《大宅门》，能给展示一个就好了。

**Reply by tact520** [reply]
我来给各位解释一下这个东西怎么用的:

在存储里,首先你需要添加几个其他存储.比如我添加了/阿里云盘   /百度网盘,这两个存储

这个时候我想要把它们合并成一个文件夹,那么我就要在存储里点添加,驱动选择"别名"

挂载路径填一个你想要显示出来的新文件夹,比如我写了/Media

路径是最重要的,我把我的给你们参考:

本地:/阿里云盘/Media
本地:/百度网盘/Media

这样设置就会把我存储里的阿里云盘/Media文件夹,与百度网盘里的/Media文件夹合并成一个新的/Media文件夹了,规则上面是提到的

**Reply by tact520** [reply]
回错位置了不好意思,好像也不能删就是了...

**Reply by tact520** [reply]
不过按照我这个设置方法,子文件夹和孙文件夹是可以合并的,只要文件名一样就会合并掉

**Reply by haohao232323** [reply]
> 不过按照我这个设置方法,子文件夹和孙文件夹是可以合并的,只要文件名一样就会合并掉

还是感谢回复，网盘有大量重复文件，不知道有没有好办法，批量清理一下。例如：周杰伦-七里香，七里香-周杰伦，七里香

**Comment by 952362** [comment]
使用别名存储作为游客访问，发现如果在别名存储中使用下载功能会失败，通过访问确认别名会改变目录路径导致请求全部失效
比如：我有一个文件夹，他的目录是 infinicloud/Apps/Books/，当我把这个文件路径加入别名存储中，在别名中打开任意文件选择下载，此时会自动跳转一个新的路径 p/infinicloud/Apps/Books/文件名，该问题会导致别名请求全部无效，因为请求的路径全是错的，请问这个可以修复吗

**Reply by meteor614** [reply]
同问
